app/javascript/src/components/common-prop-types/organization-prop-type.js

```diff
@@ -3,4 +3,6 @@ import PropTypes from 'prop-types';
 export const organizationPropType = PropTypes.shape({
   id: PropTypes.number.isRequired,
   name: PropTypes.string.isRequired,
+  slug: PropTypes.string.isRequired,
+  profile_image_90: PropTypes.string.isRequired,
```

Comment:
What's the 90 mean?

Reply:
It's from the model, but it means 90 pixels wide as far as I know.

Reply:
it's the width of the image, it comes from around the code:

```shell
➜  devto git:(master) rg profile_image_90 | wc -l
      75
```

I think originally comes from:

```ruby
ProfileImage.new(user).get(width: 90)
```